### PR TITLE
Shawnhill121 (feature)half empty heart

### PR DESCRIPTION
**Pull request by shawnhill121**
**Description**
This adds Half-Empty Heart Icons to the existing Icon set, and also to the Index.html page and Storybook Icons.

**Compatibility**
This adds a new half empty heart icon, based on the existing one. This does not alter any existing icons.

**Caveats**
None.

### CODE REVIEW

File: docs/icons.stories.js [hunk 1]
```diff
@@ -38,6 +39,9 @@ stories.add('icon', () => {
     'is-small': 'is-small',
     'is-medium': 'is-medium',
     'is-large': 'is-large',
+    'is-half': 'is-half',
```
Comment (evexoio) [inline docs/icons.stories.js]: thanks for your PR 😍

42-43 only apply to the heart icon, so they're more an icon than a variation.

Can we add `is-half` and `is-empty` after line 20 as normal icons?

File: index.html [hunk 2]
```diff
@@ -165,6 +165,16 @@ <h2 class="title">Form</h2>
           </div>
         </div>
       </div>
+      <div>
```
Comment (evexoio) [inline index.html]: I think this change doesn't belong to this PR, right?

Can we remove it please?

File: docs/icons.stories.js (hunk 1 above)
Comment (shawnhill121) [reply docs/icons.stories.js]: Thanks for the feedback, I'll take care of that. Sorry it's been a little jacked up, I'm used to using git in TFS for version control.

File: index.html (hunk 2 above)
Comment (shawnhill121) [reply index.html]: Sorry, I removed it from this one.

**Comment by guastallaigor** [comment]
Closing this because of #387.
